src/resources/views/crud/columns/date.blade.php

```diff
@@ -5,10 +5,15 @@
 
 <span data-order="{{ $value }}">
     @if (!empty($value))
-	{{
-		\Carbon\Carbon::parse($value)
-		->locale(App::getLocale())
-		->isoFormat($column['format'] ?? config('backpack.base.default_date_format'))
-	}}
+    @php($text = \Carbon\Carbon::parse($value)
+                    ->locale(App::getLocale())
+                    ->isoFormat($column['format'] ?? config('backpack.base.default_date_format')))
+        @if(isset($column['anchor']['href']))
```

Comment:
I see this ```@if(isset($column['anchor']['href']))``` statement is repeated across all columns. And as far as I can tell, it's there only because sometimes we echo using ```{{ }}```, other times using ```{!! !!}```. Correct me if I'm wrong.

Instead of having this if/else in all columns we could:
1. Move the if/else statement to ```crud::inc.column_anchors```;
2. Have an extra parameter on the column that defines what kind of echoing we want - escaped or unescaped. By default most columns would have ```'escaped' => true```, with a few columns (ex: ```model_function```, ```model_function_attribute```, ```closure```) setting ```$column['escaped'] = false;``` at the top of the column;

This would:
- make all columns look cleaner (1 line instead of 5):
```diff
-    @if(isset($column['anchor']['href']))
        @include('crud::inc.column_anchors',['text' => $text])
-    @else
-        {{$text}}
-    @endif
```

- make it easier for us to change stuff in the future (one place, instead of each column individually);
- it would create a simple file structure for columns - their only responsibility is to put together the ```$text``` variable - the actual echoing is done in the ```crud::inc.column_anchors``` file;

Reply:
Thanks for the tip. It's what I end up doing. :)